src/azure-cli/azure/cli/command_modules/security/_help.py

```diff
@@ -59,6 +59,32 @@
         az security alert update -g "myRg" --location "centralus" -n "alertName" --status "activate"
 """
 
+helps['security atp'] = """
+type: group
+short-summary: View and manage Advanced Threat Protection settings.
+"""
+
+helps['security atp show'] = """
+type: command
+short-summary: Display Advanced Threat Protection settings for a storage account.
+examples:
+  - name: Retrieve Advanced Threat Protection settings for a storage account on a subscription scope.
+    text: >
+        az security atp show --resource-group "example-rg" --storage-account-name "accountName"
+"""
```

Comment:
Could you follow Upper Camel Case Naming Notation to name option value?
Like `MyResourceGroup`, `MyAccountName`.

Reply:
Storage account name can be only in lowercase. So actually, I need to fix my Example. 
And for the resource group name - I will change the example to be as you suggested. Thanks!

Reply:
Sorry, my bad, the value here just placeholder

https://github.com/Azure/azure-cli/blob/ec3f1cbab9cb7552179ccc27f8323aff214ad02f/src/azure-cli/azure/cli/command_modules/network/_help.py#L44-L49

Please follow this pattern, remove double quotes.

Reply:
Resolved this. I checked and the CLI is case-insensitive so I changed everything according to your example.